notebooks/responsibleaidashboard/responsibleaidashboard-housing-decision-making.ipynb

```diff
@@ -226,7 +226,7 @@
    "id": "3a6d610b",
    "metadata": {},
    "source": [
-    "See this [developer blog](aka.ms/raidashboardblog) (Decision Making Flow section) to learn more about this use case and how to use the dashboard to debug your housing price prediction model."
+    "See this [developer blog](https://aka.ms/raidashboardblog) (Decision Making Flow section) to learn more about this use case and how to use the dashboard to debug your housing price prediction model."
```

Comment:
Should we put the actual blog link here instead of the aka.ms link?

https://techcommunity.microsoft.com/t5/ai-machine-learning-blog/responsible-ai-dashboard-a-one-stop-shop-for-operationalizing/ba-p/3030944

Reply:
isn't the aka.ms link actually better since we can change the link more easily if the blog moves or if we have a newer version of it?

Reply:
using Microsoft link probably brands this OSS project as a Microsoft owned project. I personally don't have a strong preference on the link used.

Reply:
Oh good point.  I'll change it to be a direct URL then.

Reply:
done, changed it to use the techcommunity link directly